src/ios/PushPlugin.m

```diff
@@ -21,9 +21,17 @@
  LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE
  OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
  ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ 
+ 
+ 
+ ADAPTED for experteer, example push
+ 
+ {"aps":{"alert":"Hello from APNs Tester.","badge":1,"sound":"default","category" : "actions"},"p":"/career/profile"}
+ 
  */
 
 #import "PushPlugin.h"
+#import "Constants.h"
```

Comment:
Hi Sascha,

Could you clarify: where is the  `Constants.h` file?  

As I understand it's not a part of the plug-in?  If that's the case it will cause problem when we try to install this plug-in in a new app?